FirebaseTestProject/Models/Post.swift

```diff
@@ -58,4 +92,16 @@ struct PostService {
     static func upload(_ post: Post) async throws {
         try await postsReference.document(post.id.uuidString).setData(post.jsonDict)
     }
+    
+    static func delete(post: Post) async throws {
```

Comment:
LMK: Would `delete(_:)` be better, or do you think there’s a benefit to explicitly naming the parameter in the method signature?

Reply:
100% `delete(_:)` -- I ended up changing it in my other branch - definitely need to do it here!